### ISSUE TITLE
PCA for extracted feature?

### ISSUE THREAD

**Issue by jiujing23333** [wzmsltw/BSN-boundary-sensitive-network.pytorch#9]
Thanks for your impressive work. I note that the feature extracted by TSN is 3072-dim but your default input dimension is 400. Have you used PCA for dimensionality reduction?

**Comment by wzmsltw** [comment]
Hi @jiujing23333 , we extract the final-layer of TSN (the classification layer), thus the dimension is 200 +200 since activitynet have 200 action classes

**Comment by jiujing23333** [comment]
Oh~I see. But for THUMOS 2014,  should I retrain a TSN？

**Comment by wzmsltw** [comment]
Of course you can fineture TSN on UCF-101, but it is also okay to directly use models pretrained on ActivityNet or Kinetic to extract feature.

**Comment by jiujing23333** [comment]
OK, many thanks.

**Comment by wzy1845** [comment]
> OK, many thanks.

Hello!  Have you successfully used TSN to extract features? Can you share the code of feature extraction? Thank you very much!!!
